### DISCUSSION TITLE
Add a `having` clause

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#1339] [Ideas]
Discussed in https://github.com/piccolo-orm/piccolo/discussions/979

https://www.postgresql.org/docs/current/tutorial-agg.html

We should also consider adding a `GroupByRaw`, similar to `OrderByRaw`, `WhereRaw` etc to handle edge cases.

**Comment by bymoye** [comment]
Hi, is there any progress?

**Comment by dantownsend** [comment]
This will be in the next release.
